helm/zeebe-test/values.yaml

```diff
@@ -1,35 +1,27 @@
 ph-ee-engine:
```

Comment:
Test helm chart can be generic like ph-ee-test instead of specific like zeebe/kafka test.

Reply:
@avikganguly01 This cannot be renamed from zeebe-test to ph-ee-test. Since we have created another chart ph-ee-test? Do you want me to rename zeebe-test to ph-ee-test and remove ph-ee-test?